Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -413,21 +413,22 @@ public State[] GetStates()
 
             return new State[0];
         }
+
         #endregion InspectorHelpers
 
         #region MonoBehaviorImplementation
 
         protected virtual void Awake()
         {
-
             if (States == null)
             {
+                // TODO: Troy - Editor only code*
```

Comment:
I am still addressing all TODO: Troy...I just wanted to post PR to get things started. 

Although the majority of files in this PR are not code to review, it is still a large change